src/main/resources/main.css

```diff
@@ -822,3 +822,15 @@ VBox.spaced, HBox.spaced {
 .error-textfield {
     -fx-effect: dropshadow(three-pass-box, red, 8, 0.0, 0, 1);
 }
+
+.selectable-text,
+.selectable-text .text-area,
+.selectable-text .text-area .scroll-pane,
+.selectable-text .text-area .viewport,
+.selectable-text .text-area .content {
+    -fx-background-color: #4d4d4d;
```

Comment:
let the color at least depend on the `.own` and `.other` class.
And I don't think you need to use that many selectors when you set the color. The topmost should be enough to lay over all other elements.